server/app/routes/api/v1/va/triplers.js

```diff
@@ -77,6 +77,13 @@ async function createTripler(req, res) {
   return res.json(serializeTripler(new_tripler));
 }
 
+async function findTripler(req, res) {
+  let found = null;
+  found = await req.neode.first('Tripler', {first_name: req.query.firstName, last_name: req.query.lastName});
```

Comment:
we need to find all triplers which match both first name and last name if entered, match first name if only first name was entered, match last name if only last name was entered